webui/react/src/services/decoder.ts

```diff
@@ -262,10 +263,21 @@ export const decodeExperimentList = (data: Sdk.V1Experiment[]): types.Experiment
   return data.map(decodeV1ExperimentToExperimentItem);
 };
 
+const filterNonScalarMetrics = (metrics: types.RawJson): types.RawJson | undefined => {
+  if (typeof metrics !== 'object') return undefined;
```

Comment:
**suggestion:** there is an `isObject` in the same place as `isNumber` (utils/data) that does it a bit more reliably. This is because `typeof null` will also report `object`.